.gitlab/deploy_6/nix.yml

```diff
@@ -1,10 +1,4 @@
 ---
-.setup_rpm_signing_key: &setup_rpm_signing_key
-  - set +x
-  - RPM_GPG_KEY=$(aws ssm get-parameter --region us-east-1 --name $RPM_GPG_KEY_SSM_NAME --with-decryption --query "Parameter.Value" --out text)
-  - printf -- "$RPM_GPG_KEY" | gpg --import --batch
-  - export RPM_SIGNING_PASSPHRASE=$(aws ssm get-parameter --region us-east-1 --name $RPM_SIGNING_PASSPHRASE_SSM_NAME --with-decryption --query "Parameter.Value" --out text)
-
 # deploy debian packages to apt staging repo
 deploy_staging_deb-6:
```

Comment:
Nit: I think these jobs should be renamed? They don't deploy anything to the staging repository anymore.

Reply:
What would you suggest? I don't really have better ideas (plus, would that mean introducing a new stage because these jobs no longer logically belong under deploy?). I totally agree that the comment above this line doesn't make sense any more, I'll remove that.

Reply:
I'd do something like `deploy_packages_deb-a6` to both remove the staging mention, and make this in line with the naming of the containers jobs.

Regarding the stage, this could go either way  they could still belong to the `deploy-7` / `deploy-6` jobs (since from the `datadog-agent` point of view, this is where we send the artifacts), or they could be in a new stage to specify that they don't deploy to the public repos.

But we can discuss this later, this does not need to block this PR.

Reply:
Right, I was thinking you meant to change the `deploy` part, not realizing that it's the `staging` part we should change because I skimmed over it :facepalm:. What you propose makes perfect sense and obviously we can leave the jobs in this stage. Thanks!